test/Npgsql.Tests/Replication/PgOutputReplicationTests.cs

```diff
@@ -47,7 +48,7 @@ public Task Insert()
                     await c.ExecuteNonQueryAsync($"INSERT INTO {tableName} (name) VALUES ('val1'), ('val2')");
 
                     using var streamingCts = new CancellationTokenSource();
-                    var messages = SkipEmptyTransactions(rc.StartReplication(slot, new PgOutputReplicationOptions(publicationName), streamingCts.Token))
+                    var messages = SkipEmptyTransactions(rc.StartReplication(slot, new PgOutputReplicationOptions(publicationName, 1), streamingCts.Token))
```

Comment:
For full test coverage, we could run the entire PgOutputReplicationTests suite twice, with the two protocol versions (like we parameterize other test suites for multiplexing yes/no)... V2-specific tests could go in another test suite.

Reply:
The only downside is, that we always need a query that creates a transaction that contains more than 64kB of payload data to actually trigger streaming.